forgerock-authenticator/src/main/java/org/forgerock/android/auth/biometric/BiometricAuth.java

```diff
@@ -1,230 +0,0 @@
-/*
- * Copyright (c) 2022 ForgeRock. All rights reserved.
- *
- * This software may be modified and distributed under the terms
- * of the MIT license. See the LICENSE file for details.
- */
-
-package org.forgerock.android.auth.biometric;
-
-import static android.content.Context.KEYGUARD_SERVICE;
-import static androidx.biometric.BiometricManager.Authenticators.BIOMETRIC_WEAK;
-import static androidx.biometric.BiometricManager.Authenticators.DEVICE_CREDENTIAL;
-
-import android.app.KeyguardManager;
-import android.content.Context;
-import android.hardware.fingerprint.FingerprintManager;
-import android.os.Build;
-
-import androidx.annotation.NonNull;
-import androidx.annotation.RestrictTo;
-import androidx.biometric.BiometricManager;
-import androidx.biometric.BiometricPrompt;
-import androidx.core.content.ContextCompat;
-import androidx.fragment.app.FragmentActivity;
-
-import org.forgerock.android.auth.Logger;
-
-import java.util.concurrent.Executor;
-
-/**
- * Helper class for managing Biometric Authentication Process.
- */
-@RestrictTo(RestrictTo.Scope.LIBRARY)
-public class BiometricAuth {
```

Comment:
I'm concerned about removing this class from the Authenticator SDK. This potentially can break existing apps... or am I missing something?

Reply:
i think we answered this, but lets test the authenticator app part of this device binding QA

Reply:
👍🏻